encoders/image/TirgImageEncoder/text_model.py

```diff
@@ -0,0 +1,126 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```

Comment:
Is it possible to remove this file? Or it fails?

Reply:
It is needed :(